porting/debug-build/udev.rst

```diff
@@ -0,0 +1,9 @@
+Add udev rules
+==============
+
+Now that you're logged in, you must create some udev rules to allow some tests to access your hardware. Run the following command, replacing [codename] with your device's codename.::
+
+    cat /var/lib/lxc/android/rootfs/ueventd*.rc|grep ^/dev|sed -e 's/^\/dev\///'|awk '{printf "ACTION==\"add\", KERNEL==\"%s\", OWNER=\"%s\", GROUP=\"%s\", MODE=\"%s\"\n",$1,$3,$4,$2}' | sed -e 's/\r//' >/etc/udev/rules.d/70-[codename].rules
```

Comment:
While at it, can we update this instructions to update udev rules on "host" machine?

ueventd*rc files are already in $OUT/root/ , so it needs updating for path and then instead of writing it to /etc/udev/rules.d/70-codename.rules, writing it to local file and then adb pushing it while in recovery..

Reply:
Actually i will not block this review for this... but todo item neverthless.

Reply:
I'm quite sure how to do this right now. That would take me some time. Let me add the todo now so we don't forget and someone can fix it later